hypervisor/debug/uart16550.c

```diff
@@ -14,20 +14,25 @@
 
 #define MAX_BDF_LEN 8
 
+union uart_addr {
+	uint16_t pio_base_paddr;
+	uint32_t mmio_base_vaddr;
+};
```

Comment:
Better to wrap serial_port_mapped together too.

Reply:
OK